File: GrantManagement2/ClientApp/src/app/Models/ApplicantGrantDetails.ts
export interface IApplicantGrantDetails {
 
  ApplicantName: string,
  ProgramCode: string,
  Country: string,
  ApplicationStatus: string,
  ReviewStatus: number

}

export class ApplicantGrantDetails implements IApplicantGrantDetails {

  ApplicantName: string;
  ProgramCode: string;
  Country: string;
  ApplicationStatus: string;
  ReviewStatus: number;


  constructor() {

  }

}

File: DataAccess/Models/ApplicantGrantDetails.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace GrantManagement.Models
{
    public class ApplicantGrantDetails
    {
        public int ApplicantId { get; set; }
      //  public int? GrantId { get; set; }
        public string FullName { get; set; }

        public string ProgramCode { get; set; }
        public string Country { get; set; }
        public string ApplicationStatus { get; set; }
        public int ReviewStatus { get; set; }



    }
}
